subprojects/docs/src/docs/release/notes.md

```diff
@@ -172,9 +147,72 @@ Using a `FileCollection` provides a `FileCollectionDependency`.
 This allows Java and Kotlin to properly configure all types of dependencies
 and improves IDE support for the Groovy DSL.
 
+In addition, the Kotlin DSL now supports named arguments for external dependencies:
```

Comment:
Should we change this section title to reflect this additional information?

Reply:
I do not think that is necessary. But I am happy to take suggestions from @big-guy or @octylFractal

Reply:
I agree, I think it works as-is.